### PR TITLE
Preparing for v2.0.2

### PR DESCRIPTION
**Pull request by rjlohan**
*Issue #, if available:* N/A

*Description of changes:* Need to release a new version with some recent changes; specifically #287 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>software.amazon.cloudformation</groupId>
     <artifactId>aws-cloudformation-rpdk-java-plugin</artifactId>
-    <version>2.0.0</version>
+    <version>2.0.2</version>
```
Comment (wbingli) [inline pom.xml:6]: why not 2.0.1?  We haven't released 2.0.1 yet,also for the Python plugin. We can release both with same version.

Comment (wbingli) [reply pom.xml:6]: I create other PR to release maven package to 2.0.1:  https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/289

Comment (rjlohan) [reply pom.xml:6]: Ah for some reason I thought we were up to 2.0.1. I'll kill this, can you please coordinate the release?

Comment (wbingli) [reply pom.xml:6]: The 2.0.1 is just released. https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/releases/tag/v2.0.1

**Pull request closed without merging**